src/benchmarks/src/end_to_end_bench.cpp

```diff
@@ -4,12 +4,11 @@
 #include <random>
 #include <vector>
 
-#include <boost/thread.hpp>
```

Comment:
Can you remove all reference to `boost/thread`? I just grepped it, and there are still a bunch in the source code:

```
./src/libclipper/include/clipper/containers.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/datatypes.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/query_processor.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/task_executor.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/threadpool.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/timers.hpp:#include <boost/thread.hpp>
./src/libclipper/include/clipper/util.hpp:#include "boost/thread.hpp"
./src/libclipper/src/containers.cpp:#include <boost/thread.hpp>
./src/libclipper/src/metrics.cpp:#include <boost/thread.hpp>
./src/libclipper/src/persistent_state.cpp:#include <boost/thread.hpp>
./src/libclipper/src/query_processor.cpp:#include <boost/thread.hpp>
./src/libclipper/src/query_processor.cpp:#include <boost/thread/executors/basic_thread_pool.hpp>
./src/libclipper/src/rpc_service.cpp:#include <boost/thread.hpp>
./src/libclipper/src/task_executor.cpp:#include <boost/thread.hpp>
./src/libclipper/src/timers.cpp:#include <boost/thread.hpp>
./src/management/src/management_frontend.hpp:#include <boost/thread.hpp>
./src/management/src/management_frontend_tests.cpp:#include <boost/thread.hpp>
```